fms_yaml_tools/schema/validate_schema.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/python3
+import click
+import yaml
+import json
+from jsonschema import validate, ValidationError, SchemaError
+
+@click.command()
+@click.option('--debug/--no-debug', type=click.BOOL, show_default=True, default=False,
+   help="Print steps in the validation")
+@click.option('--success/--no-show-success', type=click.BOOL, show_default=True, default=False,
+   help="Print success message")
+@click.argument("ypath")
+@click.argument("spath")
```

Comment:
I think the schema should be added to this repo and not in FMS because the schema is used by fms_yaml_tools not by FMS

Reply:
The schema is documentation for the code, so it should be stored with the code.  We can bring this up again at the FMS meeting if you want.